pkg/kubelet/volumemanager/cache/actual_state_of_world_test.go

```diff
@@ -318,6 +328,15 @@ func Test_AddPodToVolume_Negative_VolumeDoesntExist(t *testing.T) {
 			volumeSpec,
 			err)
 	}
+
+	blockplugin, err := volumePluginMgr.FindMapperPluginBySpec(volumeSpec)
```

Comment:
Why does this test need to FindMapperPlugin but the others don't?

Reply:
Since fake volume plugin has both NewMounter and NewBlockVolumeMapper methods, FindxxxBySpec doesn't need in other methods.

But I thought this method is trying to test FindPluginBySpec using volumeSpec.
Because volumetesting.GetTestVolumePluginMgr returns fake volume plugin, but line 301 is intentionally masking plugin from return. Therefore, I also added FindMapperPluginBySpec() to obtain plugin from volumeSpec.
Is this make sense?

Reply:
Do we have test cases where block plugin is nil? since the majority of volume plugins are not going to implement the block interface yet.

Reply:
I'll add test cases to check plugin nil case.